invenio_rdm_migrator/load/postgresql.py

```diff
@@ -32,7 +32,7 @@ def as_csv_row(dc):
         val = getattr(dc, f.name)
         if val:
             if issubclass(f.type, (dict,)):
-                val = json.dumps(val)
+                val = json.dumps(val, default=str)
```

Comment:
Yes, if you see in line 38 we are already doing it for those outside of dicts. Since it is the only case where that appeared I think is a bit overkill to implement a decoder, but can go that way. Alternative, making this func recursive when there are dicts.

Reply:
We can shelve it as well for now, it's not super critical if it's only one place (but if see the same issue come up again, we should "do it proper" :) )